src/components/KEmptyState/KEmptyState.vue

```diff
@@ -40,43 +40,39 @@
   </section>
 </template>
 
-<script lang="ts">
-import { defineComponent } from 'vue'
+<script lang="ts" setup>
 import KButton from '@/components/KButton/KButton.vue'
 import KIcon from '@/components/KIcon/KIcon.vue'
 
-export default defineComponent({
-  name: 'KEmptyState',
-  components: { KButton, KIcon },
-  props: {
-    isError: {
-      type: Boolean,
-      default: false,
-    },
-    iconSize: {
-      type: String,
-      default: '50',
-    },
-    icon: {
-      type: String,
-      default: '',
-    },
-    ctaIsHidden: {
-      type: Boolean,
-      default: false,
-    },
-    ctaText: {
-      type: String,
-      default: '',
-    },
-    handleClick: {
-      type: Function,
-      default: null,
-    },
-    iconColor: {
-      type: String,
-      default: '',
-    },
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const props = defineProps({
```

Comment:
`defineProps` is a compiler macro, so if you don't need to access the props in the setup function, you don't need to set it to a const

```suggestion
defineProps({
```

Reply:
fixed